pkgs/data/fonts/windows-fonts/default.nix

```diff
@@ -0,0 +1,85 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  p7zip,
+  # To select only certain languages, put a list of strings to `languages`: every key in
+  # ./${pname}-languages.nix is an optional language
+  languages ? [ ],
```

Comment:
It's more recommand to use `outputs`, see: https://ryantm.github.io/nixpkgs/stdenv/multiple-output/